modules/aruco/src/aruco.cpp

```diff
@@ -810,19 +810,30 @@ static void _identifyCandidates(InputArray _image, vector< vector< vector< Point
 
 
 /**
-  * @brief Return object points for the system centered in a single marker, given the marker length
+  * @brief Return object points for the system centered in a top left corner of single marker, given the marker length
```

Comment:
Need to cover all function parameters. Center coordinate system is default behaviour, but not the only one.

Reply:
fixed